### ISSUE TITLE
Webpack pined to below 4.33 (only)... why?

### ISSUE THREAD

**Issue by maraisr** [seek-oss/treat#79]
Is there an appetite to upgrade this project to allow consumers to use more modern webpack versions?

Upgrading to `webpack@latest` or any `~4.40` seems to produce this error:
```
Error: No dependency factory available for this dependency type: SingleEntryDependency
```

I can't find any changelogs from webpack that to me would indicate a change to break Treat. Maybe ya'll know of something that treat would be using from Webpack that could possibly be deprecated?

#### World:
 - storybook 5.2.8
 - treat@1.1.7
 - react-treat@1.1.7
 - react@16.12

**Comment by maraisr** [comment]
cc @markdalgleish @mattsjones @michaeltaranto

**Comment by maraisr** [comment]
further that; noticed when using webpack@4.3x, the webpack logs push out things like, if this helps at all...


```
Child treat-webpack-loader:
    Entrypoint undefined = useBoxStyles.treat
```
> concerned about that "undefined"

**Comment by mattsjones** [comment]
treat works with any version of webpack above `4.30.0`. 

```
Error: No dependency factory available for this dependency type: SingleEntryDependency
```
This error is due to having multiple versions of webpack in node_modules. Unfortunately, this can be a bit hard to manage when you are using many webpack based tools (storybook, playroom, etc). I'm sorry it's such a cryptic error message.

**Comment by maraisr** [comment]
Do you have any idea how sorry I am about this @mattsjones!?.... I know I personally hate it when people complaining about a dependency, and not the lib itself - and here I am, complaining about something other than treat... So am mega sorry for that. @mattsjones  if there is anything i can do to rectify the situation, please do let me know. 

As soon as you said, multiple versions; I ran a `yarn list webpack`, and saw the garbage.... Look thanks for this! Issue is fixed! Good old `resolutions: { webpack: 4.41.5 }` ✅
